### ISSUE TITLE
More candidates for NWaves logo ))

### ISSUE THREAD

**Issue by ar1st0crat** [ar1st0crat/NWaves#21]
![logos](https://github.com/ar1st0crat/NWaves/blob/master/screenshots/logo/candidates.png?raw=true)

**Comment by bayesrule** [comment]
1 or 2.
2 at first glance. (Probably improve a bit on 2? the NW in the background is too transparent to me; the foreground "spectrogram" is a bit too dense, but I really like the colour)

Well, just some personal feelings. =)

**Comment by ar1st0crat** [comment]
Yeah, I also like the *idea* of [2] )) And I agree, its realization could be better (there's something that makes it look less cool than it should, but I just can't figure out what exactly )))

**Comment by andy840119** [comment]
![image](https://user-images.githubusercontent.com/9100368/82721361-7ad73200-9cf7-11ea-8e76-8e40639c1f60.png)
I think [1] is better to display in small size(like nuget package).

**Comment by andrasfuchs** [comment]
Probably my vote doesn't count much, because I'm not a contributor (yet), but I like [1] the most.

**Comment by ar1st0crat** [comment]
@andrasfuchs Everyone's vote counts! )))
This issue is open for all users and fans ).

**Comment by rna0** [comment]
As yet another regular user (would like to contribute) I like 2 the most cause I've never seen one quite like it, but there is a need to simplify it a bit for the nuget icon
